composer/trainer/trainer.py

```diff
@@ -433,6 +434,11 @@ class Trainer:
             no algorithms will be used. (default: ``None``)
 
             .. seealso:: :mod:`composer.algorithms` for the different algorithms built into Composer.
+        algorithm_passes ([AlgorithmPass | Tuple[AlgorithmPass, int] | Sequence[AlgorithmPass | Tuple[AlgorithmPass, int]], optional):
+            Optional list of passes to change order in which algorithms are applied. These passes are merged with already
+            specified passes in :class:`.Engine`. If ``None``, then no passes will be used. (default: ``None``)
```

Comment:
Suggest being a little clearer here about what merging is happening.
"Optional list of passes to change the order in which algorithms are applied. These passes are merged with the default passes specific in :class:`.Engine`. If ``None``, no extra passes will be merged with the default passes.